.github/workflows/cd.yml

```
@@ -0,0 +1,37 @@
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download JAR from CI workflow
+        uses: dawidd6/action-download-artifact@v4
+        with:
+          workflow: "Java CI with Gradle"
+          name: application
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)